hip-more-cultural-names/common/landed_titles/0_HIP_MoreCulturalNames.txt

```diff
@@ -29377,11 +29274,6 @@ e_carpathia = {
                 b_kozarac = {
                 }
                 b_vrbaski_grad = {
-                    bosnian = "Vrba�ki Grad"
```

Comment:
why

Reply:
The default name is already the same: "b_vrbaski_grad;Vrbaški Grad;;;;;;;;;;;;;x"